AutomatedTesting/Gem/PythonCoverage/Code/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 #
 #
 
-ly_get_list_relative_pal_filename(pal_dir ${CMAKE_CURRENT_LIST_DIR}/Platform/${PAL_PLATFORM_NAME} ${o3de_gem_restricted_path} ${o3de_gem_path} ${o3de_gem_name})
+o3de_pal_dir(pal_dir ${CMAKE_CURRENT_LIST_DIR}/Platform/${PAL_PLATFORM_NAME} ${o3de_gem_restricted_path} ${o3de_gem_path} ${o3de_gem_name})
```

Comment:
So this file is using "**o3de**_gem_restricted_path", "**o3de**_gem_path" and "**o3de**gem_name", but the next file of AutomatedTesting/Gem/PythonTests/CMakeLists.txt is only using "gem_restricted_path", "gem_path" and "gem_parent_relative_path"